src/Core/Framework/DataAbstractionLayer/Entity.php

```diff
@@ -9,6 +9,7 @@
 use Shopware\Core\Framework\Struct\Struct;
 
 #[Package('core')]
+#[AllowDynamicProperties]
```

Comment:
we should not allow dynamic properties. instead the missing ones should be added